src_c/display.c

```diff
@@ -820,11 +826,36 @@ int _get_video_window_pos(int *x, int *y, int *center_window)
     return 0;
 }
 
+static int SDLCALL
+pg_ResizeEventWatch(void *userdata, SDL_Event *event) {
+    SDL_Window *pygame_window = pg_GetDefaultWindow();
+    PyObject *self= (PyObject *) userdata;
+    _DisplayState *state = DISPLAY_MOD_STATE(self);
+
+    if (pg_renderer!=NULL)
+        return 0;
+
+    if (state->using_gl)
+        return 0;
+
+    if (event->type == SDL_WINDOWEVENT && event->window.event == SDL_WINDOWEVENT_SIZE_CHANGED) {
+        SDL_Window *window = SDL_GetWindowFromID(event->window.windowID);
+
+        if (window == pygame_window) {
+            SDL_Surface *sdl_surface = SDL_GetWindowSurface(window);
+            pgSurfaceObject *old_surface = pg_GetDefaultWindowSurface();
```

Comment:
```sugestion
            pgSurfaceObject *old_surface = (pgSurfaceObject *)pg_GetDefaultWindowSurface();
```
(at least, I see a compiler warning about incompatible types from that line)

Reply:
I remember now. I had changed the return type of `pg_GetDefaultWindowSurface()`, but I wanted to ask here if there's a reason why it's not (pgSurfaceObject *) and maybe make that into its own commit so I can revert it easily.

I'll just add the casts for now.

Reply:
I added these type changes back in, and it compiles.